modules/test/playwright/pages/portal-workflow-kaleo-designer-web/WorkflowDefinitionPage.ts

```diff
@@ -0,0 +1,90 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Locator, Page} from '@playwright/test';
+
+import {getRandomInt} from '../../utils/util';
+import {ApplicationsMenuPage} from '../product-navigation-applications-menu/ApplicationsMenuPage';
+import {TimerPage} from './TimerPage';
+
+export class WorkflowDefinitionPage {
+	readonly addTimerButton: Locator;
+	readonly applicationsMenuPage: ApplicationsMenuPage;
+	readonly backButton: Locator;
+	readonly inputTaskTimerName: Locator;
+	readonly page: Page;
+	readonly reviewNodeLink: Locator;
+	readonly singleAproverWorkflowDefinitionLink: Locator;
+	readonly timerPage: TimerPage;
```

Comment:
I think that `timerPage` shouldn´t be a direct member of `WorkflowDefinitionPage`. I you think about the hierarchy of pages in workflow, we should have something like `WorkflowDefinitionPage` -> `DiagramViewPage` -> `NodePropertiesSidebarPage` -> `TimerPage`. Structuring like this from the get go will ease the addition of more pages in the future.